src/main/java/core/db/StatisticQuery.java

```diff
@@ -8,31 +8,68 @@
 import core.util.HODateTime;
 import core.util.HOLogger;
 
+import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.Timestamp;
 import java.time.temporal.ChronoUnit;
 import java.util.*;
+import java.util.stream.Collectors;
 
 
 public class StatisticQuery {
 
+	private static String getPlaceholders(int anzahl){
```

Comment:
That could be put in a util class  and used in that sample code I posted above!

Reply:
Also, could you please rename `anzahl` to `count`?  I get what it means, but not everyone would! ;-)

Reply:
moved it to DBManager.